control-plane/p4RuntimeSerializer.cpp

```diff
@@ -938,6 +940,7 @@ class P4RuntimeSerializer {
             auto paramName = controlPlaneName(actionParam);
             param->set_id(index++);
             param->set_name(paramName);
+            addAnnotations(param, annotations);
```

Comment:
this is incorrect, it picks up the action's annotations, not the parameter's annotations